_observing-your-data/query-insights/query-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Query metrics
+parent: Query insights
+nav_order: 20
+---
+
+# Query metrics
+
+Query metrics involves comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type. Open Telemetry (oTel) is the instrumentation framework. The telemetry data can be consumed using oTel metrics [exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters).
+
+## Configuring query metrics
```

Comment:
Here and in the following sentence, it reads as though we're either configuring a tool named query metrics or literally configuring individual metrics. Is either of those our intended meaning? Do we mean something like "Configure query metric generation"?